extension/js/common/lang.ts

```diff
@@ -2,8 +2,14 @@
 
 'use strict';
 
+import { isFesUsed } from './shared.js';
```

Comment:
looks like referencing AcctStore from Lang causes cyclic dependencies (on settings/inbox page etc.)?
There is this strange error: `ReferenceError: Cannot access 'Api' before initialization`
Perhaps, it's too heavy to reference `AcctStore` from a util class like lang.ts?
We'd better introduce `isFesUsed` function as always returning `false` first, and then inject the correct function upon some initialization?
Or, better yet, and more efficient, keep a cache dictionary of boolean values per acctEmail, and add entries to this dictionary when appropriate, @tomholub?

Reply:
Too heavy, you're right.

Let's have the methods accept `isFesUsed` directly as argument, instead of acctEmail. Then settle this problem at callsite? Wherever we call this, `AcctStore` will be available. Since we already pull information from `AcctStore` in the beginning of every `View`, it's easy to add `fesUrl` to the list of fetched items and maybe store `isFesUsed` as a private property on that `View` class.